src/Symfony/Component/Security/Core/User/ChainUserProvider.php

```diff
@@ -104,4 +104,16 @@ public function supportsClass($class)
 
         return false;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function upgradePassword(UserInterface $user, string $newEncodedPassword): void
+    {
+        foreach ($this->providers as $provider) {
+            if ($provider instanceof PasswordUpgraderInterface) {
```

Comment:
The same occurs here, should it really fails silently ?

Reply:
yes it should: you don't want your users not being able to log in when upgrades cannot happen.